src/Handler/JsonContentHandler.php

```diff
@@ -1,36 +0,0 @@
-<?php
-
-namespace Equip\Handler;
-
-use Equip\Exception\HttpException;
-use Relay\Middleware\JsonContentHandler as AbstractHandler;
-
-/**
- * @deprecated 1.4.0 Switched to Relay.Middleware
- */
-class JsonContentHandler extends AbstractHandler
-{
-    /**
-     * @inheritDoc
-     */
-    public function __construct($assoc = true, $maxDepth = 512, $options = 0)
```

Comment:
Actually this is not compatible with Relay's version. We'll have to keep this or our `$input` parsing will not work correctly (mixing object and array type).